app/models/game.rb

```diff
@@ -0,0 +1,10 @@
+class Game < ApplicationRecord
+  belongs_to :user
+  has_many :rounds
+  validates :game_mode, presence: true
```

Comment:
can you also check for specific game options?
for example game mode can only be:
-Try Out
-MQA
-Sortable

Reply:
added